File: python/8 kyu/35.py
# https://www.codewars.com/kata/570a6a46455d08ff8d001002/train/python

# Passed

def no_boring_zeros(n):
    if not n:
        return n
    
    return int(str(n).rstrip('0'))

output = no_boring_zeros(10450)
print(output)

File: nodejs/7 kyu/35.js
// https://www.codewars.com/kata/58356a94f8358058f30004b5/train/javascript

// Passed

function flyBy(lamps, drone) {
  if (drone.length >= lamps.length) return "o".repeat(lamps.length);
  const onLamps = "o".repeat(drone.length);
  const offLamps = "x".repeat(lamps.length - drone.length);
  return onLamps.concat(offLamps);
}

const output = flyBy("xxxxxx", "====T");
console.log(output);